Sources/Orbit/Components/Card.swift

```diff
@@ -192,6 +186,39 @@ public extension AccessibilityID {
     static let cardAction               = Self(rawValue: "orbit.card.action")
 }
 
+// MARK: - Types
+
+/// The layout of Orbit ``Card`` stack content.
+public enum CardLayout {
+    
+    /// Content layout uses a `VStack` with customizable padding and spacing.
+    case custom(alignment: HorizontalAlignment = .leading, padding: CGFloat = .medium, spacing: CGFloat = .medium)
```

Comment:
Do you anticipate more `case`s here? If not, there isn't really a reason for this to be an `enum`.

Reply:
I was thinking about future cases like `lazyStack` (which would use iOS14+ LazyVStack for both stacks that are currently used) etc. But it makes sense to keep it as a struct for now.

Reply:
Replaced with a struct with two public predefined values (`default` and `fill`)